CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - A new `webWorkerScript` asset type that points to the web worker script. The complete URL (base URL + URL suffix) needs to be passed as a `String` via the assets map to the `SetupData`.
 - The version of XaynAI runs in a web-worker.
+- Every method/static function of `XaynAI` throws a `TimeoutException` if the web worker does not respond after 15 seconds. (see [#290](https://github.com/xaynetwork/xayn_ai/pull/290))
```

Comment:
"after" sounds a bit like the worker is expected to respond once the time has passed
```suggestion
- Every method/static function of `XaynAI` throws a `TimeoutException` if the web worker does not respond within 15 seconds. (see [#290](https://github.com/xaynetwork/xayn_ai/pull/290))
```